frontend/src/lib/components/InsightLegend/InsightLegend.tsx

```diff
@@ -13,20 +13,15 @@ export interface InsightLegendProps {
     inCardView?: boolean
 }
 
-const trendTypeCanShowLegendDenyList = [
+export const trendTypeCanShowLegendDenyList = [
     ChartDisplayType.WorldMap,
     ChartDisplayType.ActionsTable,
     ChartDisplayType.BoldNumber,
     ChartDisplayType.ActionsBarValue,
 ]
 
-const insightViewCanShowLegendAllowList = [InsightType.TRENDS, InsightType.STICKINESS]
-
 export const shouldShowLegend = (filters: Partial<FilterType>): boolean =>
-    insightViewCanShowLegendAllowList.includes(filters.insight || InsightType.TRENDS) &&
-    isFilterWithDisplay(filters) &&
```

Comment:
`isFilterWithDisplay(filters)` == `isTrendsFilter(filters) || isStickinessFilter(filters)`, thus the above line was redundant.